docs_zh-CN/tutorials/training_tricks.md

```diff
@@ -1 +1,51 @@
-# 教程 5: 训练小技巧
+# 教程 5: 训练技巧
+
+MMSegmentation 支持如下训练技巧：
+
+## 主干网络和头组件使用不同的学习率 (Learning Rate, LR)
+
+在语义分割里，一些方法会让头组件的学习率大于骨架组件的学习率，这样可以获得更好的表现或更快的收敛。
```

Comment:
```suggestion
在语义分割里，一些方法会让头组件的学习率大于主干网络的学习率，这样可以获得更好的表现或更快的收敛。
```

Reply:
We may replace them all.

Reply:
> We may replace them all.

OK, I will replace all of it in every single doc.